deploy/helm/sumologic/conf/traces/traces.otelcol.conf.yaml

```diff
@@ -12,5 +12,6 @@
 {{- $excludePodRegex := .Values.fluentd.logs.containers.excludePodRegex | quote }}
 {{- $excludeContainerRegex := .Values.fluentd.logs.containers.excludeContainerRegex | quote }}
 {{- $excludeHostRegex := .Values.fluentd.logs.containers.excludeHostRegex | quote }}
+{{- $clusterName := .Values.sumologic.clusterName | replace " " "-" }}
```

Comment:
To have consistency with logs and metrics, you should use this one as logs and metrics use this one - https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/helm/sumologic/templates/_helpers.tpl#L317

Reply:
Fair point! I attempted doing that originally but chart had some errors while rendering, though now I see it was a simple mistake and the helper attribute works here. Thank you!